webcomponents/tool/src/main/frontend/packages/sakai-grader/src/sakai-grader-rendering-mixin.js

```diff
@@ -232,6 +227,18 @@ export const graderRenderingMixin = Base => class extends Base {
                 </span>`;
   }
 
+  _renderGraderLinkBlock() {
+
+    return html`<div id="grader-link-block" class="float-end">
```

Comment:
This could be formatted a little better, like the other template functions. Making the div an obvious block makes it read easier, imo.

Reply:
You should write down your preferred template style so we can follow it.  I looked through the file, and it looks like you indent start-of-new-tag 2 spaces and attributes for spaces.  And start of long tag is always on a new line after the html back quote bit so we can match start / end.  I reformatted it as what seemed to be the dominate approach in the file :)